components/gitpod-db/src/typeorm/migration/1646803519382-PrebuildUpdatableSHA.ts

```diff
@@ -0,0 +1,21 @@
+/**
+ * Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License-AGPL.txt in the project root for license information.
+ */
+
+import {MigrationInterface, QueryRunner} from "typeorm";
+import { columnExists } from "./helper/helper";
+
+export class PrebuildUpdatableSHA1646803519382 implements MigrationInterface {
+
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        if (!(await columnExists(queryRunner, "d_b_prebuilt_workspace_updatable", "commitSHA"))) {
+            await queryRunner.query("ALTER TABLE d_b_prebuilt_workspace_updatable ADD COLUMN commitSHA varchar(255) NOT NULL");
```

Comment:
What happens if you add a `NOT NULL` column to an existing table, without a default value? 🤔

Maybe this should be `NULL`, since `commitSHA` can be `undefined` in `DBPrebuiltWorkspaceUpdatable`.

Reply:
addressed in https://github.com/gitpod-io/gitpod/pull/8711